lib/theme/hs_color.dart

```diff
@@ -1,9 +1,8 @@
 part of 'hs_theme.dart';
 
 class HSColor {
-  static const Color primary = Color(0xFF32A854); // new design color
```

Comment:
Please keep the comment of new design, until we completely change all colours to new design then we can remove them